[15.0][ADD] website_sale_payment

dsolanki-initos opened the pull request:
This module depends on "stock_picking_on_hold".
PR: https://github.com/OCA/sale-workflow/pull/2073

dsolanki-initos committed:
[15.0][ADD] Website Sale payment

rousseldenis commented:
@dsolanki-initos Is it a real migration or an addition ?

You should enhance the module description as it does not reflect what it does really.

Moreover it depends on website_sale module, IMHO the module name should take this into account too

dsolanki-initos commented:
> @dsolanki-initos Is it a real migration or an addition ?
> 
> You should enhance the module description as it does not reflect what it does really.
> 
> Moreover it depends on website_sale module, IMHO the module name should take this into account too

Hello @rousseldenis ,
It is an addition not the migration.
I've added more description about this module. Also updated module name considering dependant module.

jans23 commented on wesite_sale_payment/__manifest__.py:
```diff
@@ -0,0 +1,23 @@
+    "name": "Website Sale Payment",
+    "version": "15.0.1.0.0",
+    "category": "Website",
+    "author": "ERP Harbor Consulting Services,"
```
Add Nitrokey GmbH

dsolanki-initos replied:
Hello @jans23 

I've already added and you can check this URL for the same. 

https://github.com/OCA/sale-workflow/blob/3a8c23e24f1cacfdae3f8c0fec8776f72f2514df/wesite_sale_payment/__manifest__.py#L5-L8

Regards,

Dhara Solanki

dsolanki-initos commented:
@AaronHForgeFlow Can you please review this PR?

dsolanki-initos commented:
@MateuGForgeFlow Can you please review this PR?

AaronHForgeFlow commented:
Can you update the PR title according to the new name? BTW I think the name is too generic, this modules has a very specific function.

dsolanki-initos commented:
> Can you update the PR title according to the new name? BTW I think the name is too generic, this modules has a very specific function.

I've changed the PR title

dsolanki-initos commented:
@CarlosRoca13 Can you please review this PR?

ikapasi-initos reviewed (approved):
Code Review and functional test.
LGTM

dsolanki-initos commented:
@omar7r Could you please review this PR?

dsolanki-initos committed:
[CHANGE] add stock_picking_on_hold in website_sale_payment's dependancy

CarlosRoca13 commented:
The name of the module is so generic, and does not reflect the intentionality of the module.

In addition, it needs to be checked why the tests are not passing and the runboat is not being built. Please check this before mentioning it to all reviewers, to avoid noise in their inbox.
